lighthouse-core/lib/i18n/locales/en-XA.json

```diff
@@ -1,117 +1,90 @@
 {
   "lighthouse-core/audits/accessibility/accesskeys.js | description": {
-    "message": "[Åççéšš ķéýš ļéţ ûšéŕš qûîçķļý ƒöçûš å þåŕţ öƒ ţĥé þåĝé. Föŕ þŕöþéŕ ñåvîĝåţîöñ, éåçĥ åççéšš ķéý mûšţ бé ûñîqûé. ᐅ[ᐊĻéåŕñ möŕéᐅ](https://web.dev/accesskeys/)ᐊ. one two three four five six seven eight nine ten eleven twelve thirteen fourteen fifteen sixteen seventeen eighteen nineteen twenty]"
+    "message": "[Åççéšš ķéýš ļéţ ûšéŕš qûîçķļý ƒöçûš å þåŕţ öƒ ţĥé þåĝé. Föŕ þŕöþéŕ ñåvîĝåţîöñ, éåçĥ åççéšš ķéý mûšţ бé ûñîqûé. [Ļéåŕñ möŕé](ĥţţþš://ðéqûéûñîvéŕšîţý.çöm/ŕûļéš/åxé/3.1/åççéššķéýš¿åþþļîçåţîöñ=ļîĝĥţĥöûšé). one two three four five six seven eight nine ten eleven twelve thirteen fourteen fifteen sixteen seventeen eighteen nineteen twenty twentyone twentytwo twentythree twentyfour twentyfive twentysix twentyseven]"
```

Comment:
`ĥţţþš://ðéqûéûñîvéŕšîţý.çöm/ŕûļéš/åxé/3.1/åççéššķéýš¿åþþļîçåţîöñ=ļîĝĥţĥöûšé` This is how you know it's definitely messed up, links should never be accented in en-XA.